ilc/package.json

```diff
@@ -8,6 +8,7 @@
     "test:ci": "npm run test -- --forbid-only",
     "test:watch": "npm run test -- --watch",
     "test:coverage": "cross-env NODE_ENV=test nyc mocha",
+    "test:coverage-text": "cross-env NODE_ENV=test nyc --reporter=text mocha",
```

Comment:
Why would you need it?

Reply:
@StyleT Probably that might be useful for someone to check the coverage quickly and be sure about whether they have covered all mandatory lines.

I'm using that for analysis to be sure whether I have missed covering some conditions/lines or maybe that doesn't make sense in some context.

![image](https://user-images.githubusercontent.com/7149640/119526425-0ca0d180-bd88-11eb-9651-277fe827f856.png)

Reply:
Could you instead change config of the `npm run test:coverage` so it would produce text report alongside with html? Should fulfil your needs, enhance CI output while ensuring that we don't have "unused" scripts.

Reply:
as for me, it looks like a useful reminder to cover all files with tests :)

Reply:
@StyleT agreed.
Just was concerning to have some redundant output upon CI building.

I've made a new commit with replaced text reporter output into default coverage reporting configuration.
Could you please have a look